File: double_loop/double_loop.ts
console.log(Deno.args[0]);
const MAX : number = parseInt(Deno.args[0]);

let num: number = 0;
for (let i: number = 0; i <= MAX; i++) {
    for (let j: number = 0; j <= MAX; j++) {
        num += j;
        num = num & 0xfff;
    }
}

console.log(num);

File: double_loop/double_loop.rs
use std::env;

fn main() {
    let max : i32 = env::args().nth(1).expect("error").parse().unwrap();

    let mut i : i32 = 0;
    let mut num : i32 = 0;
    while i <= max {
        let mut j : i32 = 0;
        while j <= max {
            num += i;
            num = num & 0xfff;
            j += 1;
        }
        i += 1;
    }

    println!("{}", num);
}